.github/workflows/engine_pr_test.yml

```diff
@@ -101,6 +101,11 @@ jobs:
           source ~/openquake/bin/activate
           pip install pytest https://wheelhouse.openquake.org/v3/py/rtgmpy-1.0.0-py3-none-any.whl
           oq engine --upgrade-db
+          oq webui start 127.0.0.1:8800 -s &
+          echo "Waiting WEBUI up on port 8800...."
+          while ! nc -z localhost 8800; do
+           sleep 5 # wait for 1/10 of the second before check again
```

Comment:
Wouldn't it sleep for 5 seconds?
We might replace the line with
`sleep 5  # seconds`